test/functional/ui/float_spec.lua

```diff
@@ -9389,6 +9389,179 @@ describe('float window', function()
         })
       end
     end)
+
+    it('ensures correct zindex order when using vim.schedule', function()
```

Comment:
can the existing test be modified? why do we need a separate test?  https://github.com/neovim/neovim/blob/e36e68d35cfa285b28f4c87782311e913a306c92/test/functional/ui/float_spec.lua#L8388

Reply:
screen height is needed to make the position of the two windows clear in the test. Special trigger condition vim.schedule

Reply:
why can't we just modify the existing test with those changes?

Reply:
by "modify the existing test", I don't mean to paste the entire new test in the existing test scope. I mean just change the existing test. Why do we need 200 lines of test code? Just change the _existing_ test for z-index behavior so that it uses `vim.schedule` and whatever else.

Reply:
because of some line wrap. existing tests are the same and each window uses a new config. prediction code for the two-window screen is  long.

Reply:
maybe need reuse and clean . trying..